configuration/assigning-devices.md

```diff
@@ -11,71 +11,121 @@ redirect_from:
 Assigning Devices to VMs
 ========================
 
+Sometimes you may need to assign an entire PCI or PCI Express device directly
+to a qube.
+This is also known as PCI pass-through.
+The Qubes installer does this by default for `sys-net` (assigning all network class controllers),
+as well as `sys-usb` (assigning all USB controllers) if you chose to create the
+USB qube during install.
+While this covers most use cases, there are some occasions when you may want to
+manually assign one NIC to `sys-net` and another to a custom NetVM,
+or have some other type of PCI controller you want to manually assign.
+
+Note that one can only assign full PCI or PCI Express devices by default.
+This limit is imposed by the PC and VT-d architectures.
+This means if a PCI device has multiple functions, all instances
+of it need to be assigned to the same qube unless you have disabled FLR with the
+`no-strict-reset` (R4.0) or `pci_strictreset` (R3.2) option.
+In the steps below, you can tell if this is needed if you see the BDF for the
+same device listed multiple times with only the number after the "." changing.
+
+While a device can only be attached to one VM at a time, it *is* possible to
+*assign* the same device to more than one VM at a time. 
```

Comment:
Should we define "assigned" and "attached" somewhere additionally? This is very subtle difference and it can be confusing in some cases.

Reply:
Edited, hopefully more clear now.